Dockerfile

```diff
@@ -1,31 +1,37 @@
 FROM ruby:2.7.2-alpine3.12
 
+ARG UID=1001
+
 RUN apk add --update nodejs yarn build-base bash libcurl git tzdata
 
-RUN addgroup -g 1001 -S appgroup && \
-  adduser -u 1001 -S appuser -G appgroup
+RUN addgroup -g ${UID} -S appgroup && \
+  adduser -u ${UID} -S appuser -G appgroup
 
 WORKDIR /app
 
-COPY Gemfile* .ruby-version ./
-
-ARG BUNDLE_FLAGS
-RUN gem install bundler
-RUN bundle install --jobs 2 --retry 3 --no-cache ${BUNDLE_FLAGS}
+RUN chown appuser:appgroup /app
 
-COPY . .
-
-ADD https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem ./rds-ca-2019-root.pem
-ADD https://s3.amazonaws.com/rds-downloads/rds-ca-2015-root.pem ./rds-ca-2015-root.pem
+ADD --chown=appuser:appgroup https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem ./rds-ca-2019-root.pem
+ADD --chown=appuser:appgroup https://s3.amazonaws.com/rds-downloads/rds-ca-2015-root.pem ./rds-ca-2015-root.pem
 RUN cat ./rds-ca-2019-root.pem > ./rds-ca-bundle-root.crt
 RUN cat ./rds-ca-2015-root.pem >> ./rds-ca-bundle-root.crt
+RUN chown appuser:appgroup ./rds-ca-bundle-root.crt
 
-RUN chown -R 1001:appgroup /app
-USER 1001
+COPY --chown=appuser:appgroup Gemfile* .ruby-version ./
+
+RUN gem install bundler
+
+RUN bundle config set no-cache 'true'
+ARG BUNDLE_ARGS='--jobs 2 --retry 3 --without test development'
+RUN bundle install ${BUNDLE_ARGS}
+
+COPY --chown=appuser:appgroup . .
 
 ENV APP_PORT 3000
 EXPOSE $APP_PORT
 
+USER ${UID}
+
 ARG RAILS_ENV=production
 RUN yarn install --production --check-files
```

Comment:
i think this is where we need to do `RUN ./bin/webpack`

Reply:
Happy to do it or pair on it, although why we don't need to do the same thing in the editor?